packages/terra-abstract-modal/src/terra-dev-site/doc/example/_AbstractModalCommon.scss

```diff
@@ -0,0 +1,7 @@
+:local {
```

Comment:
Lets set a single background-color for these examples and then remove `AbstractModalIsOpen.scss` and `AbstractModalIsFullscreen.scss` and `AbstractModalCloseOnOutsideClick.scss`.

Reply:
Done